### PR TITLE
ESU component

### LINKED ISSUE #70: ESU component
Screenshot 2019-07-31 at 14 17 32

### PR DESCRIPTION
**Pull request by LouiaR**
Type: feat

Fixes #70 

ESU header

### CODE REVIEW

File: src/components/Atoms/Link/Link.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import StyledLink from './Link.style';
 
 const Link = ({ children, color, href, target, type, home, ...rest }) => {
   const window = target === 'blank' ? '_blank' : '_self';
-  const relationship = target === 'blank' ? 'noopener noreferrer' : false;
+  const relationship = target === 'blank' ? 'noopener noreferrer' : 'false';
```
Comment (gusliedke) [inline src/components/Atoms/Link/Link.js]: Let's remove `rel` altogether. `rel="false"` is not correct. I think the only place using `rel` is our `logo` link. Logo should add this attribute.

Comment (LouiaR) [reply src/components/Atoms/Link/Link.js]: Ok sure

File: src/components/Molecules/EmailSignUp/EmailSignUp.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    <ESUWrapper {...rest}>
+      {isSubscriptionSuccessful}
+      <PrivacyCopyWraper>
+        <Text tag="p">
+          Our
+          <PrivacyLink href={privacyURL}>Privacy Policy</PrivacyLink>
+          describes how we handle and protect your information.
+          <br />
+          <br />
+          If you are under 18, please make sure you have your parents permission
```
Comment (gusliedke) [inline src/components/Molecules/EmailSignUp/EmailSignUp.js]: Use another paragraph.

Comment (LouiaR) [reply src/components/Molecules/EmailSignUp/EmailSignUp.js]: Done

**Pull request merged**